### DISCUSSION TITLE
Attempt to have bienici.com website in dark mode

### DISCUSSION THREAD

**Discussion by oeloo** [darkreader/darkreader#9793] [Q&A]
Hi, bienici.com website is in light mode despite DarkReader being active.
To fix this I have tried to add this in the dynamic theme editor following [instructions ](https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md) but the website is still staying in light mode

================================
bienici.com

CSS
body {
    background-image: none !important;
    background-color: var(--darkreader-neutral-background) !important;
}

**Comment by Gusted** [comment]
Please use [the issues](https://github.com/darkreader/darkreader/issues/new/choose) tab to report such broken websites.
